src/Tape.hpp

```diff
@@ -195,12 +195,27 @@ struct PlaybackState
 		nodes.clear();
 	}
 
+	template<typename T>
+	T* getPtr(const YAML::Node& offsetYamlNode)
+	{
+		assert(fileMmap != nullptr);
+		auto offset = offsetYamlNode.as<size_t>();
+		if (offset > mmapSize) {
+			throw std::runtime_error(fmt::format("Tape binary offset ({}) out of range ({})", offset, mmapSize));
+		}
```

Comment:
Isn't it supposed to be `>=`? I'd expect `offset` can be [0, mmapSize)

Reply:
You are right.
More precisely, shouldn't it be `[0, mmapSize - sizeof(T)]`?